stdlib/Random/src/generation.jl

```diff
@@ -167,6 +167,20 @@ function rand(r::AbstractRNG, ::SamplerType{T}) where {T<:AbstractChar}
     (c < 0xd800) ? T(c) : T(c+0x800)
 end
 
+### random tuples
+
+function Sampler(RNG::Type{<:AbstractRNG}, t::Type{T}, n::Repetition) where {T <: Tuple}
```

Comment:
Unless someone is assertive that it will be equivalent performance wise, I'd rather have this:
```suggestion
function Sampler(::Type{RNG}, ::Type{T}, n::Repetition) where {T <: Tuple} where {RNG<:AbstractRNG}
```
And use `T` instead of `t` below. It's been a long while since i checked, but i remember at least once where this would prevent type instabilities.